Source/ChocolateyGui.Subprocess/ChocolateyService.cs

```diff
@@ -205,7 +202,6 @@ public Task<bool> IsElevated()
             var operationContext = OperationContext.Current;
             using (await Lock.ReadLockAsync())
             {
-                var originalConfig = Config.get_configuration_settings().deep_copy();

```

Comment:
Does this most recent version fix the issue that necessitated this hack? Where config settings were getting carried across invokes?

Reply:
Yes, 0.10.8 has fixes for this

Reply:
Yes, 0.10.8 has fixes for this